Automattic/wp-calypso
# Launch: Send user back to original page when leaving Domains step

### Proposed Changes

Improves the behavior of the back button when the user clicks 'Launch Site'...
* From 'Settings -> General', use 'Back to General Settings' as the back link text and link them to `/settings/general/<site-slug>`.
* From 'My Home', use 'Back to My Home' as the back link text and link them to '/home/<site-slug>'.
* From the Launch Banner, use 'Back to My Site' as the back link text and link them to their primary site URL.

Previously, the link always said 'Back to My Sites' and took them to `/sites/`.

See #64211

#### After

<img width="525" alt="image" src="https://user-images.githubusercontent.com/36432/171637365-75d3b0b1-aa4e-4f5b-8698-43004b11be96.png">

#### Before

<img width="427" alt="image" src="https://user-images.githubusercontent.com/36432/171638084-2a611402-77fa-4a4f-be80-196d2612cdbf.png">

### Testing Instructions

1. Create a new Simple site.
2. Navigate to 'Settings -> General' and click on 'Launch site'.
3. Observe `&source=general-settings` added to the resulting URL.
4. Observe back link text in top left to say 'Back to General Settings'
